src/wp-includes/script-loader.php

```diff
@@ -221,9 +221,9 @@ function wp_default_packages_scripts( $scripts ) {
 	$suffix = wp_scripts_get_suffix();
 
 	// Expects multidimensional array like:
-	//	'a11y.js' => array('dependencies' => array(...), 'version' => '...'),
-	//	'annotations.js' => array('dependencies' => array(...), 'version' => '...'),
-	//	'api-fetch.js' => array(...
+	//  'a11y.js' => array('dependencies' => array(...), 'version' => '...'),
+	//  'annotations.js' => array('dependencies' => array(...), 'version' => '...'),
+	//  'api-fetch.js' => array(...
```

Comment:
Shouldn't this be in multiline comment format?

Reply:
I agree that would be better, though that would be a different fix, outside the scope of this atomic commit.
Also: WP Core does not scan against the `WordPress-Docs` standard at all yet (and rightfully so as they are nowhere near good enough yet).